### PR TITLE
Master pipeline

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
Initial master pipeline for the purpose of continuously releasing master branch to dev environment.

In its current form it does:

1. Runs tests
2. Builds and pushes docker image to dockerhub
3. Releases that image to dev env hosted by red hat openshift

Re-jira'd https://helsinkisolutionoffice.atlassian.net/browse/HAUKI-145 to run e2e tests automatically after successful release

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -1,11 +1,47 @@
...
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    #gzip  on;
```
Comment (VilleMiekkoja) [inline nginx/nginx.conf:27]: - [ ] Clean these comments and unnecessary empty lines away

File: scripts/env.sh [hunk 2]
```diff
@@ -2,12 +2,8 @@
 
 set -e -o pipefail
 
-# Recreate config file
-rm -rf ./env-config.js
```
Comment (minevala) [inline scripts/env.sh:6]: I think we still need remove the old one when we are running the script in the local env 🤔

File: release-master-to-test-environment.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Add steps that build, run tests, deploy, and more:
```
Comment (minevala) [inline release-master-to-test-environment.yml:1]: Do we need separate pipeline to make the test deploy? Or is there a way to extend the azure-pipeline.yml?

**Review by minevala — COMMENTED** [review]
Nice 🔥

File: release-master-to-test-environment.yml (hunk 3 above)
Comment (minevala) [reply release-master-to-test-environment.yml:1]: Or maybe we can group them somehow? by name? by using folder?

Comment (VilleMiekkoja) [reply release-master-to-test-environment.yml:1]: We discussed this and decided we'll put those pipeline files into a separate folder:

- [ ] Put azure pipeline files to a separate folder

**Review by minevala — APPROVED** [review]

**Pull request merged**